packages/styles/globals.import.styl

```diff
@@ -4,7 +4,7 @@ body
 
 body
   font-family inherit
-  > div
+  id['__flow-root']
```

Comment:
Yeah, you could put in another div and set it to be the root node: https://github.com/meteorhacks/flow-layout#set-different-root-node